run_configurations/test_config.py

```diff
@@ -176,9 +176,36 @@
 
     "runs": [
         {
-            "hazard": "relative_crop_yield",
-            "sectors": ["agriculture"],
-            "countries": ['Canada','Egypt', 'Portugal','Pakistan','Armenia','Belarus','Peru'],
+            "hazard": "relative_crop_yield_whe",
+            "sectors": ["agriculture_whe"],
```

Comment:
(Preface: this is potentially a nice-to-have rather than a requirement)

It's a lot of copy-pasting to specify config runs for each crop individually, and I assume we'll never want to run some crops but not others, or run different crops in different countries. I'd suggest only specifying 'relative_crop_yield' in the config and letting analysis.py do the heavy lifting of translating that into four different crop simulations (probably in the config_to_dataframe method) and re-aggregating them for the supply chain run.

Reply:
I agree, we can expand and contract automatically during the pipeline